src/transformer/defaults.ts

```diff
@@ -0,0 +1,124 @@
+// #region defaults
+interface Defaults {
+	seed: number;
+	constrained: boolean;
+	array: {
+		min: number;
+		max: number;
+	};
+	map: {
+		min: number;
+		max: number;
+	};
+	set: {
+		min: number;
+		max: number;
+	};
+	int: {
+		min: number;
+		max: number;
+	};
+	float: {
+		min: number;
+		max: number;
+	};
+	bigint: {
+		min: bigint;
+		max: bigint;
+	};
+	date: {
+		min: number;
+		max: number;
+	};
+	string: {
+		min: number;
+		max: number;
+		characterSet: string;
+	};
+}
+// #endregion defaults
+
+const DEFAULT_MIN_LIST_SIZE = 1;
+const DEFAULT_MAX_LIST_SIZE = 3;
+const MIN_SAFE_INTEGER = Number.MIN_SAFE_INTEGER;
+const MAX_SAFE_INTEGER = Number.MAX_SAFE_INTEGER;
+
+export const constrained = {
+	seed: Math.floor(Math.random() * Math.pow(10, 13)),
+	constrained: true,
+	array: {
+		min: DEFAULT_MAX_LIST_SIZE,
+		max: DEFAULT_MAX_LIST_SIZE,
+	},
+	map: {
+		min: DEFAULT_MAX_LIST_SIZE,
+		max: DEFAULT_MAX_LIST_SIZE,
+	},
+	set: {
+		min: DEFAULT_MAX_LIST_SIZE,
+		max: DEFAULT_MAX_LIST_SIZE,
+	},
+	int: {
+		min: MIN_SAFE_INTEGER,
+		max: MAX_SAFE_INTEGER,
+	},
+	float: {
+		min: MIN_SAFE_INTEGER,
+		max: MAX_SAFE_INTEGER,
+	},
```

Comment:
Should we add a constrain on these as well?

Reply:
I did -100 to +100. Does that seem reasonable?